README.md

```diff
@@ -1 +1,263 @@
-# angular-inheriting-or-isolate-scope-readme
\ No newline at end of file
+# Inheriting or isolating scope
+
+## Overview
+
+So far, our directives we've made aren't really that useful. We're not doing anything special - let's change that!
```

Comment:
Let's be a little bit more specific for the overview. Mention the problem that we're still dealing with - that we haven't seen any dynamic templates yet.